### PR TITLE
Use modern intellij-platform-plugin-template

### PR DESCRIPTION
**Pull request by stukalin**
I tried to bump the plugin version to 2021.2 and it wasn't straightforward for me because I had to upgrade the `gradle-intellij-plugin` to the latest version due to the https://github.com/JetBrains/gradle-intellij-plugin/issues/674 and it required to upgrade gradle and this whole thing didn't work anyhow. 

So I ended up grabbing the latest intellij-platform-plugin-template and adjusting it so that it basically has the same settings as we used to have. 

The IDE version bump PR comes after this one is accepted

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+fun properties(key: String) = project.findProperty(key).toString()
+
+plugins {
+    // Java support
+    id("java")
+    // Kotlin support
+    id("org.jetbrains.kotlin.jvm") version "1.5.10"
+    // gradle-intellij-plugin - read more: https://github.com/JetBrains/gradle-intellij-plugin
+    id("org.jetbrains.intellij") version "1.0"
```
Comment (stukalin) [inline build.gradle.kts:11]: I haven't upgraded it here to the latest version yet but it's still an upgrade to v1.0. 

And there are some deprecations in v1.1.4 which I wanna address first.

**Comment by Turbo87** [comment]
Thanks!

I'm currently on vacation but I'll have a look once I'm back next week.

**Comment by Turbo87** [comment]
I've rebased and slightly adjusted it. Thanks a lot for this!! :)

File: build.gradle.kts (hunk 1 above)
Comment (Turbo87) [reply build.gradle.kts:11]: @stukalin any pointers on what the next steps are?

- update the plugin to the latest version
- update the IntelliJ version
- release the plugin

like that? 😅

**Comment by stukalin** [comment]
Correct! Now it's my turn to be on holidays. Will push the next pr on Fri.

ср, 18 авг. 2021 г., 14:35 Tobias Bieniek ***@***.***>:

**Comment by Turbo87** [comment]
> Will push the next pr on Fri.

awesome, thanks



> Now it's my turn to be on holidays

enjoy :)

**Pull request merged**